packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -480,27 +479,6 @@ export class PerformanceEvent {
     }
 }
 
-// Note - these Telemetry types should move to common-definitions package
-
-/**
- * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
- * Please do not modify existing entries for backwards compatibility.
- */
-export enum TelemetryDataTag {
-    /** Data containing terms from code packages that may have been dynamically loaded */
-    PackageData = "PackageData",
-    /** Personal data of a variety of classifications that pertains to the user */
-    UserData = "UserData",
-}
```

Comment:
Keep this - the type of tag is still `string` but this provides canonical values to use.

Reply:
added back